### PR TITLE
Update readme and add esc key shortcut

### PR DESCRIPTION
**Pull request by leoarc**
## Description

1) Added esc key to go out of roi process
2) Updated readme of model and segment app

## How Has This Been Tested?
**Please describe in detail how you tested your changes.
OS :  Ubuntu 20.04
Browser : Firefox , Chromium 

## Types of changes
**What types of changes does your code introduce? Put an `x` in all the boxes that apply:
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
**Go over all the following points, and put an `x` in all the boxes that apply.
**(If you're unsure about any of these, don't hesitate to ask. We're here to help!)
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by birm** [comment]
@Insiyaa

File: apps/model/model.js [hunk 1]
```diff
@@ -1572,3 +1572,11 @@ function updateTextInput(val) {
   document.getElementById('textInput').value = val;
 }
 
+
+$(document).keydown(function(event) { 
+  if (event.keyCode == 27) { 
+    $('#roi_panel').hide();
+    $('#choice_panel').hide();
```
Comment (birm) [inline apps/model/model.js:1579]:
```suggestion
    $('#choice_panel').hide();
    $('#model_info').hide();
```

File: apps/segment/segment.js [hunk 2]
```diff
@@ -1819,3 +1819,11 @@ async function extractRoiSelect(choices) {
   drawRectangle({checked: true, state: 'roi', model: choices.model});
 }
 
+
+$(document).keydown(function(event) { 
+  if (event.keyCode == 27) { 
+    $('#roi_panel').hide();
+    $('#choice_panel').hide();
```
Comment (birm) [inline apps/segment/segment.js:1826]:
```suggestion
    $('#choice_panel').hide();
    $('#model_info').hide();
```

**Comment by birm** [comment]
Oh, we should probably test in more browsers; mozilla at least warns about depreciation for keyCode in plain js.

**Comment by leoarc** [comment]
tested with chrome and Firefox .

**Pull request merged**